klaviyo/api_helper.py

```diff
@@ -200,6 +212,25 @@ def _public_request(self, path, querystring):
         url = '{}/{}?{}'.format(self.api_server, path, querystring)
         return self._request(self.HTTP_GET, url, request_type=self.PUBLIC)
 
+    def _public_post_request(self, url, datastring):
+        """Track and identify calls, always a post request.
+
+        Args:
+            url (str): endpoint url.
+            datastring (str): to pass into data field of url-encoded form.
+        Returns:
+            (str): 1 or 0 (pass/fail).
+        """
+
+        headers = {
```

Comment:
I'm thinking something more like:
```
# class constant:
BASE_HEADERS = {
   'Content-Type': 'application/json',
    'User-Agent': 'Klaviyo-Python/{}'.format(__version__)
}
def _request(..., headers={})
...
self.BASE_HEADERS.update(headers)

```

Reply:
we'll need to make a copy of BASE then no? so we don't permanently change the default

Reply:
you want to update them if something new is passed in, update is an upsert operations